new-dti-website/components/bottom.tsx

```diff
@@ -49,9 +49,9 @@ const Bottom: React.FC = () => (
               alt="Trends icon"
             />
             <div className="flex flex-col justify-start w-full items-start">
-              <h6 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
+              <h4 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
```

Comment:
```suggestion
              <h3 className="text-left w-full font-bold md:text-sm lg:text-[22px]">
```

Then to not skip over anything we can bump the `<h4>`s to `<h3>`